src/reducers/notesReducer.js

```diff
@@ -0,0 +1,15 @@
+
+const notesReducer = (state,{type,payload}) => {
+    switch(type){
+        case "TITLE":
+            return {...state, title : payload}
+        case 'BODY':
+            return {...state,body : payload}
+        case "RESET":
+            return {...state, title : "" , body : ""}
+        default:
+            return
```

Comment:
in default you can just return the state

Reply:
Makes sense, thanks